dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/scheduling/RecordingJobProgress.java

```diff
@@ -199,7 +199,7 @@ public void startingProcess(String description, Object... args) {
     observer.run();
 
     if (isCancelled()) {
-      throw new CancellationException();
+      throw newCancellationException();
```

Comment:
when I first read it I thought this is a typo and invalid Java 😅 missing a space in between new and the exception. Maybe rename it to `cancellationException()` anyone else has the same first thought.